cfme/cloud/provider/openstack.py

```diff
@@ -85,3 +89,18 @@ def from_config(cls, prov_config, prov_key, appliance=None):
                    tenant_mapping=prov_config.get('tenant_mapping', False),
                    infra_provider=infra_provider,
                    appliance=appliance)
+
+    # Following methods will only work if the remote console window is open
+    # and if selenium focused on it. These will not work if the selenium is
+    # focused on Appliance window.
```

Comment:
All these methods will throw NoSuchElementException consistently. Do we still need to make it explicit in the code with try block?